temporalio/workflow.py

```diff
@@ -500,6 +500,14 @@ class UpdateInfo:
     name: str
     """Update type name."""
 
+    @property
+    def logger_details(self) -> Mapping[str, Any]:
```

Comment:
```suggestion
    def _logger_details(self) -> Mapping[str, Any]:
```

I don't think we should make this public (and have it then be stable API)